### PR TITLE
remove +49 from 800 numbers

### PR DESCRIPTION
**Pull request by dsarkar**
closes https://github.com/corona-warn-app/cwa-website/issues/755

### CODE REVIEW

**Review by loki-cui — APPROVED** [review]

File: src/data/faq.json [hunk 1]
```diff
@@ -317,7 +317,7 @@
                         "active": true,
                         "textblock": [
                             "The app is currently available in the following languages:<ul><li>German</li><li>English</li><li>Turkish (as of version 1.1.1)</li><li>Bulgarian (as of version 1.2.0)</li><li>Polish (as of version 1.2.0)</li><li>Romanian (as of version 1.2.0)</li></ul>",
-                            "The telephone hotline (<a href='tel:+498007540001'>+49 800 7540001</a>) answers in Turkish, English and German. More languages are to come. We'll keep you up to date.", "The language of the app is derived from the language that is configured in the smartphone's language settings. To display the app in German, for example, change the language settings to German.",
+                            "The telephone hotline (<a href='tel: 08007540001'> 0800 7540001</a>) answers in Turkish, English and German. More languages are to come. We'll keep you up to date.", "The language of the app is derived from the language that is configured in the smartphone's language settings. To display the app in German, for example, change the language settings to German.",
```
Comment (mtb77) [inline src/data/faq.json:320]: You need to remove the trailing space before 0800 so it will be tel:0800xxx

Comment (mtb77) [reply src/data/faq.json:320]: same for >0800

File: src/data/faq.json [hunk 2]
```diff
@@ -122,7 +122,7 @@
...
                             "Currently, 90% of the existing test capacities in Germany are covered by the complete digital process. We are continuing to work with the laboratories to close the gaps and to connect and upgrade all laboratories for the process. Therefore it can happen that a laboratory gets the order for the test with the correct form, but has to transmit the test result to the tested person in another way.",
-                            "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a TeleTAN by calling the hotline at +49 800 7540002 to warn your risk encounters. Please enter the TeleTAN into the Corona Warn App by following these steps:",
+                            "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a TeleTAN by calling the hotline at  0800 7540002 to warn your risk encounters. Please enter the TeleTAN into the Corona Warn App by following these steps:",
```
Comment (mtb77) [inline src/data/faq.json:125]: at 0800
instead of 
at  0800

File: src/data/faq.json [hunk 3]
```diff
@@ -517,15 +517,15 @@
...
+                            "If you have more than one smartphone (e.g. work and private), we recommend that you install the CWA on your private device first. Also, when scanning QR codes and sharing a positive result, we recommend that you first use your private device. If you additionally want to share a positive result via your company device or any other additional smartphone, this can be done via a so-called TeleTAN. By contacting  0800 7540002 with a positive test result, you will receive a 10-digit TAN with which the test result can also be registered in the app. <b>Please note:</b> The TAN number is only valid for one hour."
```
Comment (mtb77) [inline src/data/faq.json:520]: contacting 0800 
instead of 
contacting  0800

**Review by mtb77 — CHANGES_REQUESTED** [review]
Please see the comments in the PR

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:320]: ok

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json:125]: ok

**Pull request merged**